include/eredis_cluster.hrl

```diff
@@ -6,10 +6,11 @@
 
 -type redis_error_result() :: Reason::bitstring() | no_connection
     | invalid_cluster_command | tcp_closed.
--type redis_success_result() :: Result::bitstring() | undefined.
+-type redis_success_result() :: bitstring() | [bitstring()] | undefined.
```

Comment:
Where `bitstring()` (any multiple of bits) is used in these types, it should really be `binary()` (a multiple of 8 bits).

Reply:
Sounds reasonable, but I got stuck when looking into why the change was done in https://github.com/Nordix/eredis_cluster/commit/071b73be75561b370b807f909c8a8899618c951d.
Maybe it was in early development stages and was then just kept? Can an `EVAL` return anything else in Redis?

Looking at eredis v1.0.8, used at the time of the change, it states `binary()` in the `return_value()` and no `bitstring()` mentioned in the eredis code either. Sure looks like we should change this.

Reply:
EVAL can return RESP data, like any other redis command. It's not even possible to receive parts of a byte over TCP.

Possibly some other spec used bitstring() which gave some warning when it was used with binary(). Do you know if there were any occurrences of bitstring prior to that change?

Reply:
No bitstrings can be found in the repo prior to that change.